exporter/otlphttpexporter/otlp.go

```diff
@@ -223,25 +223,46 @@ func (e *baseExporter) export(ctx context.Context, url string, request []byte, p
 	formattedErr = httphelper.NewStatusFromMsgAndHTTPCode(errString, resp.StatusCode).Err()
 
 	if isRetryableStatusCode(resp.StatusCode) {
-		// A retry duration of 0 seconds will trigger the default backoff policy
-		// of our caller (retry handler).
-		retryAfter := 0
-
 		// Check if the server is overwhelmed.
 		// See spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp-throttling
 		isThrottleError := resp.StatusCode == http.StatusTooManyRequests || resp.StatusCode == http.StatusServiceUnavailable
-		if val := resp.Header.Get(headerRetryAfter); isThrottleError && val != "" {
-			if seconds, err2 := strconv.Atoi(val); err2 == nil {
-				retryAfter = seconds
+		if isThrottleError {
+			// A retry duration of 0 seconds will trigger the default backoff policy of our caller (retry handler).
+			throttleDuration := getRetryAfterSeconds(resp)
+			if throttleDuration != 0 {
+				return exporterhelper.NewThrottleRetry(formattedErr, getRetryAfterSeconds(resp))
```

Comment:
Why do we need to parse it again? 
```suggestion
				return exporterhelper.NewThrottleRetry(formattedErr, throttleDuration)
```